allennlp/nn/beam_search.py

```diff
@@ -489,7 +546,34 @@ def top_p_sampling(
             sampler=sampler_p,
         )
 
+    @classmethod
+    def stochastic_beam_search(
```

Comment:
Maybe just call this ⬇️ ?

```suggestion
    def stochastic(
```